### ISSUE TITLE
multiple choice text box smaller than its outline

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes-Front-End#621]
low in the priority queue - noticed in firefox and chrome (haven't checked in safari) that the textbox for multiple choice is much smaller than its box outline 

![screen shot 2015-06-19 at 6 58 00 pm](https://cloud.githubusercontent.com/assets/753325/8252148/5482ede2-16b5-11e5-8bf4-1cd489ec3926.png)

**Comment by mschwamb** [comment]
There doesn't appear to be a design pass on the project builder any time soon.  Any chance someone could just make the text box bigger? I found this a bit annoying for a project I just created (though isn't going through a public launch) where the responses were all 3-4 words each as the text was crowded into the left corner. I think it also gives the impression that the project builder isn't quite finished, which certainly isn't the case.
